.github/workflows/live-test.yml

```diff
@@ -1,24 +1,70 @@
 name: Live tests for B2K
 on:
-  push:
-    branches: [ "main" ]
-  pull_request:
-    branches: [ "main" ]
+  repository_dispatch:
+    types: [ok-to-test-command]
+permissions:
+  id-token: write
 jobs:
   liveTest:
+    env:
+      PR_NUMBER: ${{ github.event.client_payload.slash_command.args.named.PR_NUMBER }}
+      HEAD_REPO: ${{ github.event.client_payload.slash_command.args.named.HEAD_REPO }}
+      HEAD_BRANCH: ${{ github.event.client_payload.slash_command.args.named.HEAD_BRANCH }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, macos-latest, windows-latest]
     runs-on: ${{ matrix.os }}
     name: live test for b2k
     steps:
     - uses: actions/checkout@v2
+      # checkout PR head repo/branch
+      with:
+         repository: ${{ env.HEAD_REPO }}
+         ref: ${{ env.HEAD_BRANCH }}
+    - name: Login Azure Context
+      if: ${{ matrix.os != 'ubuntu-latest' }}
+      uses: Azure/login@v1
+      with:
+        tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+        subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+        client-id: ${{ secrets.AZURE_CLIENT_ID }}
+    - name: Prepare Cluster name Windows
```

Comment:
@cxznmhdcxz Can we combine the prepare and create AKS cluster steps for mac and windows since they are common using variable ```$RUNNER_OS```? for example 

```
- name: Prepare Cluster name Mac
      if: ${{ matrix.os == 'macos-latest' ||  matrix.os == 'windows-latest' }}
      shell: bash
      run: |
        echo "cluster_name=b2k-livetest-$RUNNER_OS-pr-$PR_NUMBER" >> $GITHUB_ENV
- name: Create AKS mac
      if: ${{ matrix.os == 'macos-latest' ||  matrix.os == 'windows-latest' }}
      run: |
        az aks create -g ${{ secrets.AZURE_RESOURCE_GROUP }} -n ${{ env.cluster_name }} --no-ssh-key

```
so that it will be 2 steps instead of 4 separate steps.